test-suite/non-functional-tests/test-scenario-1/README.md

```diff
@@ -0,0 +1,39 @@
+## Test Scenario 1
+
+12 patient transfers x 1 thread
+
+### Test details
+
+12 Transfers requested - 10 expected to succeed, 2 expected to fail, each successful transfer contains:
+* 1 RCMR_IN030000UK06 message
+* 1 COPC_IN000001UK01 index message
+* 2 COPC_IN000001UK01 Fragment messages
+
+### Prerequisites 
+
+You will need to provide the certificate and key required to connect to the inbound service.
+* Copy the certificate and key files into the /testPlans/certs folder named 'client.crt' and 'client.key' respectively.
```

Comment:
I think the path here is wrong, and in the next 5 occurances of this sentence too 🤣

Reply:
Corrected in all places